### PR TITLE
User Reservation Skeleton & Reservation Form

### PR DESCRIPTION
**Pull request by clee4037**
- Created skeleton for user reservation page
- Created address form
-- Storing address form data in 1 state {street, city, zip, ...}
-- Updated form styling to match figma

### CODE REVIEW

File: src/components/user-reservation/AddressForm.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          </div>
+          <div className="flex flex-col w-1/4">
+            <label htmlFor="apt">Apt/Unit # </label>
+            <input
+              type="text"
+              onChange={(e) => {
+                const updatedAddress = address;
+                updatedAddress.apt = e.target.value;
+                setAddress(updatedAddress);
+              }}
```
Comment (justcord10) [inline src/components/user-reservation/AddressForm.jsx:31]: I didnt even think about an apartment number until now, will need to discuss about adding this to the DB

File: src/components/user-reservation/AddressForm.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            />
+          </div>
+        </div>
+        <button type="submit">Submit</button>
+      </form>
+    </div>
+  );
+};
+
+export default AddressForm;
```
Comment (justcord10) [inline src/components/user-reservation/AddressForm.jsx:89]: good basic react skeleton, nice labels for each input to meet accessibility standards, also like the starting implementations of the color scheme. looks ready to add in states for the different input fields

File: src/components/user-reservation/Confirmation.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      <h3>Reservation Name</h3>
+      <h3>Garage</h3>
+      <h3>Address</h3>
+      <h3>Day & Time</h3>
+      <button>Confirm Reservation</button>
+    </div>
+  );
+};
+
+export default Confirmation;
```
Comment (justcord10) [inline src/components/user-reservation/Confirmation.jsx:14]: Looks ready to be set up to display input info before submission of reservations

File: src/components/user-reservation/AddressForm.jsx (hunk 1 above)
Comment (justcord10) [reply src/components/user-reservation/AddressForm.jsx:31]: after thinking about this this is fine, just remember to not Post an apartment number up to the database, but using an apartment number to find the nearest garage to that address w/ an apartment number should be fine!

**Review by justcord10 — APPROVED** [review]
skeleton looks good, just left a couple of general comments and a specific comment about your apt number location. I like the implementation of the colors as well as the accessibility standards input labels

**Review by stevkim — APPROVED** [review]
Things look good!
I like the separation of forms!

**Pull request merged**